packages/terra-search-field/src/SearchField.jsx

```diff
@@ -283,7 +284,6 @@ class SearchField extends React.Component {
             onKeyDown={this.handleKeyDown}
             onInput={this.handleInput}
             ref={this.setInputRef}
-            {...additionalInputAttributes}
```

Comment:
This has the potential to non-passively change the API and break consumers. I do think this is an appropriate change to fix the attribute priority, but it may be worth a discussion on whether or not to major version bump, treat the change as a bug fix, or move forward if we have reason to believe few people (or none) would be impacted by the change.

Reply:
I think the big thing we care about here is the className so let's keep applying any custom class names that teams might be passing in.  Anything else, I would argue we should never have been doing in the first place so (with my recommendation) I would be comfortable calling this passive.

Reply:
@saket2403 I see you resolved this conversation, but I'm not sure moving the attribute spread pulls the custom class name out to ensure custom styles arn't accidentally broken.